<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interface_didatica_menu_botao_emergencia</class>
 <widget class="QMainWindow" name="interface_didatica_menu_botao_emergencia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>841</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1290</width>
    <height>841</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1290</width>
    <height>841</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label_Borda">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>783</y>
      <width>1290</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Imagens/Borda.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_home">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>20</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-color: rgb(0, 0, 0);
border-width:5px;
font: 75 65pt &quot;Bosch Sans Bold&quot;;
border-radius: 0px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>imagens/Home.Interface.png</normaloff>imagens/Home.Interface.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>110</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_fundo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>1021</width>
      <height>761</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bosch Sans Bold</family>
      <pointsize>65</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-color: rgb(0, 0, 0);
border-width:7px;
border-radius: 0px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_imagem">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>381</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagens/botao_emergencia.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_titulo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>1021</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-color: rgb(0, 0, 0);
border-width:6px;


font: 75 35pt &quot;Bosch Sans Bold&quot;;
background-color: rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffc000;&quot;&gt;BOTÃO DE PARADA DE EMERGÊNCIA&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fundo_verde">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>75</y>
      <width>461</width>
      <height>685</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-color: rgb(0, 0, 0);
border-width:6px;
background-color: rgb(155, 228, 179);

font: 75 80pt &quot;Bosch Sans Bold&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_caixa_de_texto">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>541</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Bosch Sans Bold&quot;;

</string>
    </property>
    <property name="text">
     <string>    Quando pressionado, possui a finali-
dade de parar imediatamente o movi-
mento da máquina, desabilitando seu 
comando. O botão de parada de 
emergência deve estar em um local 
visível da máquina e sempre ao 
alcance do operador.

    O mesmo deve ser pressionado 
caso necessite de uma parada imediata, 
como em casos de acidentes de 
trabalho, ou se a máquina apresentar 
uma funcionalidade inadequada da 
qual está programada.  
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_seta_esquerda">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>640</y>
      <width>98</width>
      <height>117</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-color: rgb(0,0,0);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>imagens/Seta Preta para esquerda.png</normaloff>imagens/Seta Preta para esquerda.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_seta_dirteita">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>640</y>
      <width>96</width>
      <height>115</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-color: rgb(0,0,0);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>imagens/Seta Branca para direita.png</normaloff>imagens/Seta Branca para direita.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="shortcut">
     <string/>
    </property>
   </widget>
   <zorder>Label_Borda</zorder>
   <zorder>label_fundo</zorder>
   <zorder>botao_home</zorder>
   <zorder>label_titulo</zorder>
   <zorder>label_fundo_verde</zorder>
   <zorder>botao_seta_esquerda</zorder>
   <zorder>botao_seta_dirteita</zorder>
   <zorder>label_imagem</zorder>
   <zorder>label_caixa_de_texto</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1290</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
